build.sbt

```diff
@@ -467,4 +471,57 @@ lazy val compatibilityTests = Project("akka-http-compatibility-tests", file("akk
     }
   )
 
+
+lazy val `maven-dependencies` = Project(id="maven-dependencies",base= file("maven-dependencies"))
+  .enablePlugins(HeaderPlugin)
+  .settings(
+    name := "akka-http-maven-dependencies",
```

Comment:
is this a typical name for a BOM? the idea is it should be usable from gradle as well, right?

Reply:
Good point. A little bit of googling seems to point that `akka-http-bom` may be more appropriate. I just took what we already have on Lagom and renamed/adapted a few things.